ffmpeg/transcoder.c

```diff
@@ -86,6 +86,11 @@ void lpms_init(enum LPMSLogLevel max_level)
   av_log_set_level(max_level);
 }
 
+void lpms_init_xcoder_params(char *ts_params)
+{
+
```

Comment:
Is this function empty to implement in the future? Seams like `struct input_params.xcoderParams` contains same data.

Reply:
removed